packages/adblocker/src/filters/network.ts

```diff
@@ -727,27 +727,35 @@ export default class NetworkFilter implements IFilter {
       // --------------------------------------------------------------------- //
       // parseOptions
       // --------------------------------------------------------------------- //
+      const domainsList: Set<string> = new Set();
+      const denyallowList: Set<string> = new Set();
```

Comment:
Nit. Can we create an empty set lazily so that it's not needed for the vast majority of filters? We could set them to `undefined` by default.

Reply:
updated in 025db7c5a9c0213b5cf228e1913d2aef9e17a0e1